vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.LibreSSL

```diff
@@ -0,0 +1,7 @@
+SunnyValley: {
+  fingerprints: "/usr/local/etc/pkg/fingerprints/SunnyValley",
+  url: "https://updates.sunnyvalley.io/opnsense/LibreSSL/repo/",
+  signature_type: "fingerprints",
+  priority: 101,
```

Comment:
this asserts higher priority than OPNsense repo (11). If you ship the wrong packages it'll take yours and could bring back security risks from not in sync sources. I just want to mention it. I'm not sure it can be avoided as pkg is weird sometimes in multi-repo envs.

Reply:
Nope, not indeed. We want svn repo have a lower priority. Had a look at this again. Looks like we need to have lower values if we want lower priority. How about if I put 7 here?

Reply:
I think 11 has no significance other than raising the priority above FreeBSD which uses "0". It's a joke from Spinal Tap... "these go to 11".

So in all seriousness 7 should be fine then. Thanks!